shell/browser/api/electron_api_web_contents.cc

```diff
@@ -708,6 +708,7 @@ content::WebContents* WebContents::CreateCustomWebContents(
 void WebContents::AddNewContents(
     content::WebContents* source,
     std::unique_ptr<content::WebContents> new_contents,
+    const GURL& target_url,
```

Comment:
should we expose this anywhere?

Reply:
Looks like we are already using the `target_url` to populate the url of child window here https://github.com/electron/electron/blob/master/shell/browser/api/electron_api_web_contents.cc#L677 , so there is no need to expose it.